score-server/src/main/java/bio/overture/score/server/security/ssl/SSLCertificateValidation.java

```diff
@@ -0,0 +1,25 @@
+package bio.overture.score.server.security.ssl;
+
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+
+public class SSLCertificateValidation {
+  public static void disable() {
+    try {
+      SSLContext sslContext = SSLContext.getInstance("TLS");
+      TrustManager[] trustManagerArray = new TrustManager[] {new DumbX509TrustManager()};
+      sslContext.init((KeyManager[]) null, trustManagerArray, (SecureRandom) null);
+      HttpsURLConnection.setDefaultSSLSocketFactory(sslContext.getSocketFactory());
+      HttpsURLConnection.setDefaultHostnameVerifier(new DumbHostnameVerifier());
+    } catch (NoSuchAlgorithmException | KeyManagementException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
```

Comment:
Does this mimic the SSL Cert verification being done by DCC core? Looks like we're just trusting everything with our trust manager and verifier performing no tests...

Reply:
Yes, the SSL verification mimic the code which is there in DCC Libraries and we have migrated the code.